src/components/layout/TopBar.less

```diff
@@ -42,6 +43,27 @@
   margin-bottom: 10px;
 }
 
+.top-bar__link_skip {
+  border: 0;
+  color: var(--g3-color__white);
+  margin: 10px auto 5px 30px;
+  width: auto;
+  height: 1.7em;
+  display: block;
+  border-bottom: 1px solid var(--g3-color__white);
+  position: absolute;
+  left: -9999px;
+}
+
+.top-bar__link_skip:focus {
+  left: 0;
```

Comment:
this component will only be visible when focused by key press?

Reply:
yes that's correct, it is the standard for skip navigation buttons (as they are only used by keyboard users)